bindings/cpp/include/wtf/macros.h

```diff
@@ -44,6 +45,19 @@
 #define WTF_THREAD_ENABLE(name) \
   WTF_THREAD_ENABLE_IF(kWtfEnabledForNamespace, name)
 
+// Enables WTF tracing for any thread based on whether the current namespace
+// is enabled. Thread name will be auto-generated.
+#define WTF_ALL_THREADS_ENABLE() \
+  do { \
+    thread_local bool enabled = false; \
```

Comment:
Believe it or not, thread_local is broken/barely works on (some) Android targets. See above for a suggested approach.